Dockerfile.fedora

```diff
@@ -0,0 +1,21 @@
+# File to be used for building an Ubuntu .deb
+
+FROM fedora
+
+MAINTAINER Brenden Blanco <bblanco@plumgrid.com>
+
+RUN dnf -y install @rpm-development-tools @c-development @development-tools cmake libstdc++-static
+
+WORKDIR /root
+RUN git clone https://github.com/llvm-mirror/llvm.git
+RUN git clone https://github.com/llvm-mirror/clang.git llvm/tools/clang
+RUN mkdir -p /root/llvm/build
+WORKDIR /root/llvm/build
+RUN cmake .. -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="ARM;CppBackend;X86;BPF"
```

Comment:
With no X86, I see the following:

Native target X86 is not selected; lli will not JIT code

BPF is not shown as enabled in this case.